plugins/org.jboss.reddeer.junit.extension/src/org/jboss/reddeer/junit/extension/log/collector/LogCollector.java

```diff
@@ -100,32 +101,37 @@ public File getLogFile(String config, String className) {
 		}
 		return logFile;
 	}
-	
+
 	/**
 	 * Processes workbench log. It copies log entries from Eclipse workbench log to RedDeer log file of 
 	 * a specific test class.
 	 * 
-	 * @param config RedDeer config 
+	 * @param config RedDeer config
 	 * @param class test class name
```

Comment:
obviously I made a mistake here. Could you please parameter name to className instead of class?